templates/registration/email_change_activate_email.html

```diff
@@ -0,0 +1,24 @@
+<div style="border:2px solid #fd0;margin:4px 0;"><div style="background:#000;border:6px solid #000;">
+<a href="{{ protocol }}://{{ domain }}"><img src="{{ protocol }}://{{ domain }}/static/icons/logo.svg" alt="{{ site_name }}" width="160" height="44"></a>
+</div></div>
+
+<div style="border:2px solid #337ab7;margin:4px 0;"><div style="background:#fafafa;border:12px solid #fafafa;font-family:segoe ui,lucida grande,Arial,sans-serif;font-size:14px;">
+<br><br>
+{{ user.get_username() }},
+<br>
+{% trans %}You have requested to change your email address to this email for your user account at {{ site_name }}.{% endtrans %}
+<br><br>
+{% trans trimmed count=expiry_minutes %}
+Please click the link to confirm this email change. The link will expire in {{ count }} minute.
```

Comment:
Let's just assert that the expiry is > 1 second as part of config validation, and avoid needing to pluralize this. 1 minute expiry is unusable UX anyway.

Reply:
I've added an assertion for > 1 minute expiry, but going to wait on @quantum5's reply before changing the template to verify this will work for other languages.

Reply:
Seems like this is necessary for other languages, according to @int-y1.

Reply:
Yes, lack of ngettext is going to be so much fun in a bunch of languages...